lib/on_form/collection_wrapper.rb

```diff
@@ -2,17 +2,20 @@ module OnForm
   class CollectionWrapper
     include ::Enumerable
 
-    attr_reader :parent, :association_name, :collection_form_class, :allow_insert, :allow_update, :allow_destroy
+    attr_reader :parent, :association_name, :collection_form_class, :options
 
     delegate :each, :first, :last, :[], to: :to_a
 
-    def initialize(parent, association_name, collection_form_class, allow_insert, allow_update, allow_destroy)
+    def initialize(parent, association_name, collection_form_class, options = {})
+      default_options = {allow_insert: true, allow_update: true, allow_destroy: false, reject_if: nil}
```

Comment:
I wonder if this would be cleaner with a DEFAULT_OPTIONS constant at the top of the class instead of a local variable?